lib/seahorn/BvOpSem2Context.hh

```diff
@@ -361,6 +361,17 @@ public:
   void addToSolver(const Expr e);
   boost::tribool solve();
 
+  // If running in lambda mode,
+  // convert constant arrays returned by Z3 simplifier to a lambda.
+  //
+  // Details: When Z3 is given a lambda of the form lambda x: 0 to simplify, a
+  // constant array is returned. If running in lambda mode, this will cause the
+  // formula to have both lambdas and constant arrays which is not supported
+  // (fully) by Z3 and y2.
+  // This workaround fixes the issue by converting constant arrays gotten from
+  // Z3 back to lambda x: 0.
+  Expr convertConstArrayToLambda(Expr e);
```

Comment:
why does this need to be a class method? just a private function.

Reply:
done